.travis.yml

```diff
@@ -19,93 +20,43 @@ jobs:
       script: mdl -r "~MD013" .
 
     - stage: "Test"
-      name: "test:pycodestyle:3.6"
-      sudo: required
+      name: "test:lint"
```

Comment:
Linting belongs in stage Check IMO.
I know you didn't introduce that one, but if you agree, would you be willing to move it here?